### DISCUSSION TITLE
implement problem

### DISCUSSION THREAD

**Discussion by Confuzu** [webfp/tor-browser-selenium#190] [Q&A]
How can I tell TBS to launch the headless browser with the highest security settings. I have seen in the utils the security settings but do not get the  thing implemented in my code.

**Comment by gunesacar** [comment]
You should be able to use code from the following files to achieve that:
https://github.com/webfp/tor-browser-selenium/blob/main/examples/headless.py
https://github.com/webfp/tor-browser-selenium/blob/main/tbselenium/test/test_set_security_level.py
